ui/src/components/connections/Connections.tsx

```diff
@@ -0,0 +1,179 @@
+import { Avatar, Card, Divider, Row, Space, Table, Tag, Tooltip, Typography } from "antd";
+import { ColumnsType } from "antd/es/table";
+import { useCallback, useEffect, useState } from "react";
+import { useSearchParams } from "react-router-dom";
+
+import { APIError } from "src/adapters/api";
+import { Connection, Credential, getConnections } from "src/adapters/api/connections";
+import { ReactComponent as IconUsers } from "src/assets/icons/users-01.svg";
+import { ConnectionsRowDropdown } from "src/components/connections/ConnectionsRowDropdown";
+import { ErrorResult } from "src/components/shared/ErrorResult";
+import { NoResults } from "src/components/shared/NoResults";
+import { SiderLayoutContent } from "src/components/shared/SiderLayoutContent";
+import { TableCard } from "src/components/shared/TableCard";
+import { useEnvContext } from "src/contexts/env";
+import { isAbortedError, makeRequestAbortable } from "src/utils/browser";
+import { CONNECTIONS, QUERY_SEARCH_PARAM } from "src/utils/constants";
+import { AsyncTask, isAsyncTaskDataAvailable, isAsyncTaskStarting } from "src/utils/types";
+
+export function Connections() {
+  const env = useEnvContext();
+  const [connections, setConnections] = useState<AsyncTask<Connection[], APIError>>({
+    status: "pending",
+  });
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const queryParam = searchParams.get(QUERY_SEARCH_PARAM);
+
+  const tableContents: ColumnsType<Connection> = [
+    {
+      dataIndex: "userID",
+      ellipsis: { showTitle: false },
+      key: "type",
+      render: (userID: Connection["userID"]) => (
+        <Tooltip placement="topLeft" title={userID}>
+          <Typography.Text strong>{userID.replace(/did:\w*:\w*:\w+:/, "")}</Typography.Text>
+        </Tooltip>
+      ),
+      sorter: ({ id: a }, { id: b }) => a.localeCompare(b),
+      title: "Identifier",
+    },
+    {
+      dataIndex: "credentials",
+      ellipsis: { showTitle: false },
+      key: "credentials",
+      render: (credentials: Credential[]) => (
+        <Space>
+          {[...credentials]
+            .sort((a, b) => a.attributes.type.localeCompare(b.attributes.type))
+            .map((credential) => (
+              <Typography.Text key={credential.id}>{credential.attributes.type}</Typography.Text>
```

Comment:
The design states that the credentials need to be comma separated and this JSX is not doing that. I'd render a single Text and comma-separate the values:

```typescript
<Typography.Text>
  {credentials
    .sort((a, b) => a.attributes.type.localeCompare(b.attributes.type))
    .map((credential) => credential.attributes.type)
    .join(", ")}
</Typography.Text>
```